README.md

```diff
@@ -1,70 +1,57 @@
-# Getting Started with Create React App
+# Student App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Student App is a full-stack web application that allows users to create, edit, and delete both student and course records. It also allows users to take and drop courses for each student.
 
-## Available Scripts
+## Table of Contents
 
-In the project directory, you can run:
+- Motivation
+- Installation
+- Usage
+- API
+- Tools
+- Authors
 
-### `npm start`
+## Motivation
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+Student App was Inspired by my college's schedule builder. I wanted to create a web application that would allow students manage their courses and their schedule.
```

Comment:
```suggestion
Student App was Inspired by my college's schedule builder. I wanted to create a web application that would allow students to manage their courses and their schedule.
```